### PR TITLE
Add TAC Member R&Rs

### PR DESCRIPTION
**Pull request by afmarcum**
Proposed language for TAC Member R&Rs. This proposal pulls from Walli's P&Ps document and attempts to follow the WG Lead R&Rs format.

Open for feedback and patches. Thank you!

### CODE REVIEW

**Review by SecurityCRob — APPROVED** [review]

**Review by steiza — APPROVED** [review]

File: process/tac-member-R&Rs.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# TAC Member Roles & Responsibilities
+
+- Commit to abiding by the OpenSSF Code of Conduct
```
Comment (lehors) [inline process/tac-member-R&Rs.md:3]: While I certainly agree with the statement it applies to everybody engaging in OpenSSF, not just the TAC. Maybe it's better to leave it off and keep this list to what is specific to TAC members?

File: process/tac-member-R&Rs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Provide ongoing, technology-neutral technical guidance and sponsorship to Technical Initiatives throughout the respective lifecycle stages
+- Establish and drive an overall Technical Vision aligned with the Foundation's MVSR for the community 
+- Regularly attend TAC meetings, participate in discussions, voting, and ad-hoc meetings as needed to support Technical Initiatives
+- Approve, establish, structure, organize, and archive Technical Initiatives and associated policies and procedures 
+- Provide sponsorship and guidance to new Technical Initiatives that petition to join or be created within OpenSSF
+- Participate in Technical Initiatives regularly to ensure alignment with the Technical Vision and MVSR of OpenSSF, and to identify any resource or funding requirements
+- Coordinate such other technical community matters related to the success of Technical Initiatives and the mission of the OpenSSF
```
Comment (lehors) [inline process/tac-member-R&Rs.md:10]: "Coordinate such other"?? Something doesn't seem right here, unfortunately I'm not sure what the intent is so I can't really suggest a correction. Just remove "such"? But who is the coordination meant to be with?

**Review by lehors — CHANGES_REQUESTED** [review]
Thanks for putting this together.

File: process/tac-member-R&Rs.md (hunk 2 above)
Comment (afmarcum) [reply process/tac-member-R&Rs.md:10]: Thanks for the catch, @lehors! I was mashing together the ideas of a few other folks and grammar lost this battle!

**Review by di — APPROVED** [review]

File: process/tac-member-R&Rs.md (hunk 1 above)
Comment (afmarcum) [reply process/tac-member-R&Rs.md:3]: @presidentoor do you want to weigh in here?

Comment (presidentoor) [reply process/tac-member-R&Rs.md:3]: I think the language should read: "Abide by the OpenSSF Code of Conduct"

Comment (lehors) [reply process/tac-member-R&Rs.md:3]: Ok but the question is whether we should have any such language here at all. I think not unless we go and add it everywhere else.

Comment (lehors) [reply process/tac-member-R&Rs.md:3]: Ok but the question is whether we should have any such language here at all. I think not unless we are going to go back and add it everywhere else.

Comment (CoS-Harry) [reply process/tac-member-R&Rs.md:3]: I don't know where everywhere else is as I view this document as foundational and standalone. This document resolves into the greater set of documents anyone may read about the TAC governance. The WG R&R should surely have the same note and I think promulgating the Code Conduct far and wide through these types of documents is useful and necessary.

**Review by SecurityCRob — APPROVED** [review]

**Review by lehors — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**